web/src/components/data-entities/diver/DiverList.js

```diff
@@ -1,103 +1,129 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState, useRef} from 'react';
 import {Box, Button, Typography} from '@mui/material';
 import {NavLink} from 'react-router-dom';
 import {grey, red} from '@mui/material/colors';
 import {getResult, entityEdit} from '../../../api/api';
 import LoadingOverlay from '../../overlays/LoadingOverlay';
 import {AgGridColumn, AgGridReact} from 'ag-grid-react';
 import {Add, Save} from '@mui/icons-material';
+import {constants} from '../../../common/constants';
 import 'ag-grid-enterprise';
 
 const DiverList = () => {
-  const [gridApi, setGridApi] = useState();
-  const [delta, setDelta] = useState([]);
-  const [rowData, setRowData] = useState();
-  const [errors, setErrors] = useState([]);
+    const [delta, setDelta] = useState([]);
+    const [errors, setErrors] = useState([]);
+    const dGridRef = useRef(null);
```

Comment:
What is the naming convention for dGridRef (also oGridRef and lGridRef) ?

Reply:
Change to have name alighted